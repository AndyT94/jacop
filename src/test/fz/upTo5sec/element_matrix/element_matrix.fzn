var 1..1: index_i:: output_var;
var 3..3: index_j:: output_var;
var 7..7: value:: output_var;
solve  satisfy;
