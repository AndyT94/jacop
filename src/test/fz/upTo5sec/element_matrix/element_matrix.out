index_i = 1;
index_j = 3;
value = 7;
----------